modules/shipit_scriptworker/manifests/settings.pp

```diff
@@ -18,17 +18,19 @@
         password                  => secret('shipit_scriptworker_ship_it_password_dev'),
     }
 
-    $_env_configs             = {
+    $env_config = {
       'dev'  => {
         worker_group             => 'shipit-dev',
         worker_type              => 'shipit-dev',
         verbose_logging          => true,
```

Comment:
I think this is unused since in modules/shipit_scriptworker/manifests/init.pp you're using the one from line 111 - https://github.com/mozilla/build-puppet/pull/49/files#diff-16fc35f7191a6528caf61089d8af4f3eR111